include/glow/Support/Support.h

```diff
@@ -119,6 +119,26 @@ inline void report(llvm::StringRef str) { report(str.data()); }
 /// can be inspected under debugger.
 std::string legalizeName(llvm::StringRef name);
 
+struct MultiLineStr {
+  std::string str;
+};
+
+struct DeviceConfigHelper {
```

Comment:
Should this be in "Support.h" or some other header file that is related to device and device configurations?

Reply:
It is only used in Support.cpp for reading yaml file.